src/Core/Engine/Renderer/API/Texture2D.cpp

```
@@ -2,13 +2,26 @@
 
 #include <tracy/TracyOpenGL.hpp>
 
-void UploadTexture(unsigned char* data, int width, int height, int nrChannels)
+void UploadTexture(unsigned char* data, int width, int height, int nrChannels, bool sRGB)
```

Clang (14.0.6)

[warning] 3 adjacent parameters of 'UploadTexture' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)